<launch>
  <node name="usb_cam0" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
    <param name="framerate" value="60"/>
    <param name="contrast" value="32"/>
    <param name="brightness" value="32"/>
    <param name="saturation" value="32"/>
    <param name="sharpness" value="22"/>
    <param name="autofocus" value="false"/>
    <param name="focus" value="51"/>
  </node>
  <node name="usb_cam1" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video1" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
    <param name="framerate" value="60"/>
    <param name="contrast" value="32"/>
    <param name="brightness" value="32"/>
    <param name="saturation" value="32"/>
    <param name="sharpness" value="22"/>
    <param name="autofocus" value="false"/>
    <param name="focus" value="51"/>
  </node>
  <node name="image_view0" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam0/image_raw"/>
    <param name="autosize" value="true" />
  </node>
  <node name="image_view1" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam1/image_raw"/>
    <param name="autosize" value="true" />
  </node>
  <!--
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/color_filtered"/>
    <param name="autosize" value="true" />
  </node>
  -->
</launch>
